src/sfeQwiicUltrasonic.cpp

```diff
@@ -29,13 +40,33 @@ sfeTkError_t sfeQwiicUltrasonic::isConnected()
     return _theBus->ping();
 }
 
-sfeTkError_t sfeQwiicUltrasonic::triggerAndRead(uint16_t &distance)
+sfeTkError_t sfeQwiicUltrasonic::getDistance(uint16_t &distance)
+{
+    size_t bytesRead = 0;
+    uint8_t rawData[2] = {0, 0};
+    sfeTkError_t err;
+
+    _theBus->writeByte(kUltrasonicDistanceReadCommand);
+    err = _theBus->readRegisterRegion(_theBus->address(), rawData, 2, bytesRead);
+
+    // Check whether the read was successful
+    if (err != kSTkErrOk)
+        return err;
+
+    // Store raw data
+    distance = (rawData[0] << 8) | rawData[1];
+
+    // Done!
+    return kSTkErrOk;
+}
+
+sfeTkError_t sfeQwiicUltrasonic::getTriggeredDistance(uint16_t &distance)
```

Comment:
Renaming this breaks backwards compatibility

Reply:
Yeah this is a big one potentially, I got a bit cavalier when I changed this function name because I wasn't quite up to speed on keeping backwards compatibility. I think the only option is to put this back in, so I'll go ahead and do that.

Reply:
What's your opinion?

Reply:
Backwards compatibility is important, don't do anything to break that for existing users, unless there's a _**really**_ good reason for it.

Reply:
Fixed.